input/pagecontent/2-10-1-ContentSharingFHIRcastMessaging.md

```diff
@@ -24,7 +24,7 @@ FHIR resources are used to convey the structured information being exchanged in
 Support of content sharing by a Hub is optional.  If supporting content sharing, a FHIRcast Hub SHALL fulfill additional responsibilities:
 
 1. Assign and maintain an anchor context's `context.versionId` when processing a `[FHIR resource]-open` request - the `context.versionId` does not need to follow any convention but SHALL be unique in the scope of a topic  
-2. Reject `[FHIR resource]-update` request if the version does not match the current `context.versionId` by returning a 4xx/5xx HTTP Status Code rather than updating the content or indication of selection
+2. Reject `[FHIR resource]-update` request if the version does not match the current `context.versionId`. A request can be rejected by returning a 4xx/5xx HTTP Status Code or a 202 HTTP status code followed by a syncError.
```

Comment:
This is in a section regarding Hub responsibilities.  Given that the Hub is a "server" a response with a 4xx/5xx status code is appropriate.  In fact, the Hub SHALL (the SHALL is from the "a FHIRcast Hub SHALL fulfill" above, perhaps it should be explicit in each of the items) reject the request before propagating to Subscribers so there is no distribution of the event.  I don't believe a 202 is appropriate since it indicates a somewhat "happy" situation when a Subscriber submitting a request with the wrong versionId is definitely an "unhappy" situation.

In fact, originally I recommended getting very specific 409 Conflict status code (from Wikipedia: Indicates that the request could not be processed because of conflict in the current state of the resource, such as an [edit conflict](https://en.wikipedia.org/wiki/Edit_conflict) between multiple simultaneous updates) but I believe the collective feedback was that we should not get too specific.

I recommend this change not be made.

Reply:
Discussed on recent FHIRcast calls, re-opened and res-resolving underlying jira: https://jira.hl7.org/browse/FHIR-36974